docs/content/guide/component.ngdoc

```diff
@@ -464,8 +468,7 @@ describe('component: heroDetail', function() {
   }));
 
   it('should set the default values of the hero', function() {
-    // It's necessary to always pass the scope in the locals, so that the controller instance can be bound to it
-    component = $componentController('heroDetail', {$scope: scope});
+    component = $componentController('heroDetail', {});
```

Comment:
Nitpick: I prefer to omit the unnecessary `locals` argument altogether (or pass `null` where we want a 3rd argument).